tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -284,7 +269,8 @@ def main():
     
     failed = 0
     print("\n\tRunning s2n dynamic record size tests\n\t")
-    failed += test(host, port, test_ciphers, int(file_size / 2))
+    threshold = 10000  # threshold size set independently of file size
```

Comment:
Is it interesting to test multiple different thresholds? 

NP: it might be nice to add a comment that this threshold is number of bytes used when starting s2nc.

Reply:
Addressed in the comment in: [5ad96af](https://github.com/awslabs/s2n/pull/2292/commits/5ad96af860e216ad6e96a6e0c6253b770dceaac3)

As for different threshold sizes, this might have to be something to look into - if the threshold is made high (>20KB) it seems as if the high throughput mode comes on too early (always seems to be around the 15th packet). See this example:

![Screenshot 2020-09-17 at 14 44 19](https://user-images.githubusercontent.com/68056884/93479527-52052900-f8f4-11ea-9f5e-77a02ab54ed4.png)

I don't understand this behaviour (is this a bug in s2nc?)- this would have gone undetected in the old version of this test too - since it only looked for high throughput messages from the 18th packet onwards...

Reply:
See comment further down for resolution.